package.json

```diff
@@ -78,6 +78,7 @@
     "file-loader": "1.1.6",
     "friendly-errors-webpack-plugin": "1.6.1",
     "glob": "7.1.2",
+    "hoist-non-react-statics": "^2.3.1",
```

Comment:
Thank you, I was sporadically getting build errors related to this module but had no idea what it was about.